nav2_regulated_pure_pursuit_controller/src/parameter_handler.cpp

```diff
@@ -226,7 +226,7 @@ rcl_interfaces::msg::SetParametersResult ParameterHandler::validateParameterUpda
         logger_, "The value inflation_cost_scaling_factor is incorrectly set, "
         "it should be >0. Ignoring parameter update.");
         result.successful = false;
-      } else if (parameter.as_double() < 0.0) {
+      } else if (name.find(plugin_name_ + ".") == 0 && parameter.as_double() < 0.0) {
```

Comment:
I think as a more general purpose solution to apply to other nodes, we should check this in L222 after we get the `type` and `name` to see if the `name` is part of our node. If not, `continue;` onto the next parameter and simply bypass these checks entirely.